lib/buildpackutil.py

```diff
@@ -265,38 +275,29 @@ def _checkout_from_git_rootfs(directory, mx_version):
     )
 
 
-def fix_mono_config_and_get_env(dot_local_location, mono_lib_dir):
+def _get_env_with_monolib(mono_lib_dir):
     env = dict(os.environ)
-    env['LD_LIBRARY_PATH'] = mono_lib_dir
-
-    if not os.path.isfile(os.path.join(mono_lib_dir, 'libgdiplus.so')):
+    env['LD_LIBRARY_PATH'] = mono_lib_dir + '/lib'
+    if not os.path.isfile(os.path.join(mono_lib_dir + '/lib/', 'libgdiplus.so')):
         raise Exception('libgdiplus.so not found in dir %s' % mono_lib_dir)
-
-    subprocess.check_call([
-        'sed',
-        '-i',
-        's|/app/vendor/mono/lib/libgdiplus.so|%s|g' % os.path.join(
-            mono_lib_dir, 'libgdiplus.so'
-        ),
-        os.path.join(_get_mono_path(dot_local_location), 'etc/mono/config'),
-    ])
-    subprocess.check_call([
-        'sed',
-        '-i',
-        's|/usr/lib/libMonoPosixHelper.so|%s|g' % os.path.join(
-            mono_lib_dir, 'libMonoPosixHelper.so'
-        ),
-        os.path.join(_get_mono_path(dot_local_location), 'etc/mono/config'),
-    ])
     return env
 
 
-def _get_mono_path(dot_local_location):
+def _detect_mono_version():
+    if os.environ.get('FORCED_MONO4_VERSION'):
```

Comment:
So, this effectively means we're never going to use Mono 4 based on a Mendix version, but only if we set the `FORCED_MONO4_VERSION` env var, correct?